src/main/java/ddeliopoulos/github/gardenjournal/journalentry/JournalService.java

```diff
@@ -0,0 +1,73 @@
+package ddeliopoulos.github.gardenjournal.journalentry;
+
+import ddeliopoulos.github.gardenjournal.journalentry.api.CreateJournalRequest;
+import ddeliopoulos.github.gardenjournal.journalentry.api.GetJournalResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class JournalService {
+
+    private final JournalRepository journalRepository;
+
+    @Autowired
+    JournalService(final JournalRepository journalRepository) {
+        this.journalRepository = journalRepository;
+    }
```

Comment:
[nitpick] Could be replaced with [Lombok's `@RequiredArgsConstructor`](https://projectlombok.org/features/constructor) ⁽ᵃˡˢᵒ ˢᵉᵉ [ᵒᵘʳ ᵍᵘᶦᵈᵉˡᶦⁿᵉˢ](https://github.com/Glovo/backend-guidelines/blob/master/guidelines/lombok.md#requiredargsconstructor) ᶠᵒʳ ᵐᵒʳᵉ ᶦⁿᶠᵒ ᵒⁿ ʰᵒʷ ᵗʰᶦˢ ᵃⁿⁿᵒᵗᵃᵗᶦᵒⁿ ᶦˢ ᵘˢᵉᵈ ᶦⁿ ᵗʰᵉ ᵇᵃᶜᵏ⁻ᵉⁿᵈ⁾.

ᴛʜɪs ᴀᴄᴛɪᴏɴ ᴡᴀs ᴘᴀʀᴛɪᴀʟʟʏ ᴀᴜᴛᴏᴍᴀᴛᴇᴅ ʙʏ [GɪᴛHᴜʙ's ʀᴇsᴘᴏɴsᴇ ᴛᴇᴍᴘʟᴀᴛᴇs](https://github.com/settings/replies)